### ISSUE TITLE
[Patch Request] MC-264204

### ISSUE THREAD

**Issue by vico93** [isXander/Debugify#340]
### Mojang Bug Report ID

MC-264204

### Bug adhere to the guidelines.

- [X] I made sure that this bug adheres to the guidelines.

### Has not already been requested.

- [X] I am certain this patch has not already been requested.

### Has not already been implemented.

- [X] I know that this patch isn't already in the latest version of the mod.

### Existing implementation

https://github.com/diskree/MC-264204-Fix

**Comment by diskree** [comment]
Hi  @isXander!

I saw that you're considering adding my fix to your project. I'm really glad that my code could be useful. Since I'm not very familiar with the details of licensing, I want to make things easier for you.

So, I'm giving you permission to use my fix in your project, regardless of the MIT license I originally used. Feel free to include, modify, and distribute my code under your LGPL 3.0 license.

If you need any more info or help, just let me know.

**Comment by vico93** [comment]
> Hi @isXander!
> 
> I saw that you're considering adding my fix to your project. I'm really glad that my code could be useful. Since I'm not very familiar with the details of licensing, I want to make things easier for you.
> 
> So, I'm giving you permission to use my fix in your project, regardless of the MIT license I originally used. Feel free to include, modify, and distribute my code under your LGPL 3.0 license.
> 
> If you need any more info or help, just let me know.

Actually it was me that opened this ticket! @isXander still didnt commented here if he will add this bug on the list for the next Debugify release.

I'm using your mod already on my pack, but i would prefer having all MC-XXXXX fixes under a single mod (debugify) because my pack is already huge and i would like to reduce the amount of mods.

**Comment by diskree** [comment]
Yes, I understand that. I just saw this and decided to write it, since our projects have different licenses
